client/my-sites/checkout/composite-checkout/components/item-variation-picker/variant-dropdown-price.tsx

```diff
@@ -135,20 +135,19 @@ export const ItemVariantDropDownPrice: FunctionComponent< {
 	const productBillingTermInMonths = variant.productBillingTermInMonths;
 	const isIntroductoryOffer = introCount > 0;
 	const translate = useTranslate();
-	const billingTermInYears = () => {
-		if ( productBillingTermInMonths > 12 ) {
-			return productBillingTermInMonths / 12;
-		}
-		return null;
-	};
 
 	const translatedIntroOfferDetails = () => {
 		const args = {
 			formattedCurrentPrice,
 			formattedPriceBeforeDiscounts,
-			billingTermInYears: billingTermInYears(),
 			introCount,
 		};
+		// Add billing term in years for multi-year plans. (Only to be used when productBillingTermInMonths > 12)
```

Comment:
This can be a bit error-prone, but let's hope that the comment can help.

I've confirmed that we've used it only when the billing term in months is actually > 12.